JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md

```diff
@@ -1 +1,5 @@
-This demo illustrates how the **Chart** widget aggregates series points in&nbsp;a&nbsp;financial chart. The **useAggregation** option activates this feature. The **RangeSelector** widget zooms the contents of&nbsp;the **Chart**. Note that whenever the selected range is&nbsp;reduced, the series becomes less aggregated.
\ No newline at end of file
+The **Chart** widget can aggregate series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**. 
+
+In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. The wider you set the selection range, the higher aggregation degree the control uses, and vice versa.
```

Comment:
In the demo, are you setting the "selection" to a wide range? For example, do you mean the following:

```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. If you set the selection to a wide range, the control will use a higher aggregation degree, and vice versa.
```